service/routes.py

```diff
@@ -0,0 +1,45 @@
+import time
+import logging
+
+from flask import request
+
+from service.attack_surface import AttackSurfaceService
+from service.consts import ApiConsts
+
+logger = logging.getLogger('spam_application')
+logger.setLevel(logging.DEBUG)
+
+
+def configure_routes(app, input_file_path):
+    attack_surface = AttackSurfaceService(
+        input_file_path=input_file_path,
+    )
+    logger.info('attack surface configured properly')
+
+    @app.route('/api/v1/stats', methods=['GET'])
```

Comment:
you can use Namespace and Resource from flask_restplus
The I would have write:

@ns.route('stats')
class Stats(Resource):
    def get:

Also add doc strings for each function